CHANGELOG.rst

```diff
@@ -161,3 +178,4 @@ Bugfixes
 
 v1.0.2
 ======
+
```

Comment:
Should we add any information for v1.0.2 ?

Reply:
There isn't any information.